tasks/fix-cat2.yml

```diff
@@ -1271,12 +1274,56 @@
       - RHEL-07-040210
 
 - name: "MEDIUM | RHEL-07-040230 | PATCH | The operating system, if using PKI-based authentication, must implement a local cache of revocation data to certificate validation in case of the inability to access revocation information via the network."
-  command: "true"
+  lineinfile:
+      dest: /var/lib/pki-kra/conf/server.xml
+      regexp: ^#?auths.revocationChecking.bufferSize
+      line: auths.revocationChecking.bufferSize=50
+  when: rhel_07_040230_audit.stat.exists == True
+  tags:
+      - cat2
+      - medium
+      - patch
+      - RHEL-07-040230
+      - pki
+
+- name: "MEDIUM | RHEL-07-040230 | PATCH | The operating system, if using PKI-based authentication, must implement a local cache of revocation data to certificate validation in case of the inability to access revocation information via the network."
+  lineinfile:
+      dest: /var/lib/pki-kra/conf/server.xml
+      regexp: ^#?auths.revocationChecking.enabled
+      line: auths.revocationChecking.enabled=true
+  when: rhel_07_040230_audit.stat.exists == True
+  tags:
+      - cat2
+      - medium
+      - patch
+      - RHEL-07-040230
+      - pki
+
+- name: "MEDIUM | RHEL-07-040230 | PATCH | The operating system, if using PKI-based authentication, must implement a local cache of revocation data to certificate validation in case of the inability to access revocation information via the network."
+  lineinfile:
+      dest: /var/lib/pki-kra/conf/server.xml
+      regexp: ^#?enableOCSP
+      line: enableOCSP="true"
```

Comment:
Not sure if the quoting here will trip up the YAML parser. May need to wrap the whole line in single quotes.